strawberry/django/__init__.py

```diff
@@ -1,7 +1,7 @@
 try:
     # import modules and objects from external strawberry-graphql-django
     # package so that it can be used through strawberry.django namespace
-    from strawberry_django import *  # noqa: F401, F403
+    from strawberry_django import *  # type: ignore # noqa: F401, F403
```

Comment:
why do we need the type ignore? 😊

Reply:
mypy was unhappy because strawberry_django package was not available.

I think we have two options
* install strawberry-graphql-django package
* ignore type and make mypy happy in that way

what's your opinion?

Reply:
considering we are going to merge the packages and the issue will go away by doing that, I'd go with this solution 😊

Reply:
I'm getting this error. "fix" was to ignore it

```
/Users/l/git/graphql/strawberry/django/tests/mypy/test_dataloaders.yml:10:
E   pytest_mypy_plugins.utils.TypecheckAssertionError: Invalid output:
E   Expected:
E     main:8: note: Revealed type is 'strawberry.dataloader.DataLoader[Any, Any]' (diff)
E     main:13: note: Revealed type is 'Any'         (diff)
E   Actual:
E     ../../../../../../Users/l/git/graphql/strawberry/django/strawberry/django/__init__:4: error: Cannot find implementation or library stub for module named 'strawberry_django' (diff)
E     ../../../../../../Users/l/git/graphql/strawberry/django/strawberry/django/__init__:4: note: See https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-imports (diff)
E     main:8: note: Revealed type is 'strawberry.dataloader.DataLoader[Any, Any]' (diff)
E     main:13: note: Revealed type is 'Any'         (diff)
E
E   Alignment of first line difference:
E     E: main:8: note: Revealed type is 'strawberry.dataloader.DataLoader[Any, An...
E     A: ../../../../../../Users/l/git/graphql/strawberry/django/strawberr...
```

Reply:
I see, fine to ignore it for now :)